### PR TITLE
Fix warning/deprecated msg indentation in help text

### LINKED ISSUE #164: Marking command for preview - the preview message should be in the new line and not continue in the long summary
image

### PR DESCRIPTION
**Pull request by atbagga**
Fixes #164 

- Before the fix:
image

- After the fix:
image

### CODE REVIEW

**Comment by atbagga** [comment]
@adewaleo @tjprescott Can you please take a look at this fix?

Also the build failures seem unrelated to the change.

**Comment by adewaleo** [comment]
@atbagga we no longer work on the CLI team. Please reach out to @yonzhan 

cc: @qwordy  @Juliehzl

**Comment by yonzhan** [comment]
@jiasli please take a look at this PR.

**Comment by yonzhan** [comment]
We will plan it after Ignite.

File: knack/help.py [hunk 1]
```diff
@@ -382,9 +382,9 @@ def _build_long_summary(item):
             if item.long_summary:
                 lines.append(item.long_summary)
             if item.deprecate_info:
-                lines.append(str(item.deprecate_info.message))
+                lines.append(''.join(['\n', str(item.deprecate_info.message)]))
```
Comment (qwordy) [inline knack/help.py]:
```
lines.append('\n' + str(item.deprecate_info.message))
```
shorter

**Comment by qwordy** [comment]
The Travis CI build failed.

**Comment by qwordy** [comment]
I don't have permission to rerun CI.

**Comment by qwordy** [comment]
And it seems I don't have write access. So my approval is not sufficient.
```
Review required
At least 1 approving review is required by reviewers with write access. 
```

**Comment by atbagga** [comment]
Thanks @qwordy 
+ @jiasli For review. 

Can you also check why is the CI broken? Seems unrelated to my change.

**Comment by atbagga** [comment]
@jiasli @yonzhan CI seems to be broken for Knack. 

I tried a PR with just readme changes and yet it failed - https://github.com/microsoft/knack/pull/168

**Comment by jiasli** [comment]
@atbagga , do you mind fix the test to pass the CI? https://travis-ci.org/microsoft/knack/jobs/648761811#L274

**Comment by atbagga** [comment]
> @atbagga , do you mind fix the test to pass the CI? https://travis-ci.org/microsoft/knack/jobs/648761811#L274

Done!! @jiasli

**Comment by jiasli** [comment]
@haroldrandom , is it safe to merge to master?

**Comment by haroldrandom** [comment]
> @haroldrandom , is it safe to merge to master?

Look so. Please go ahead.

**Pull request merged**